Lib/urllib/parse.py

```diff
@@ -363,7 +394,7 @@ def _fix_result_transcoding():
 _fix_result_transcoding()
 del _fix_result_transcoding
 
-def urlparse(url, scheme='', allow_fragments=True):
+def urlparse(url, scheme='', allow_fragments=True, classes=set()):
```

Comment:
Avoid mutable defaults, an empty iterable works well. (There are other cases of this throughout the diff.)
```suggestion
def urlparse(url, scheme='', allow_fragments=True, classes=()):
```

Reply:
@JelleZijlstra using `SchemeFlag()` creates an immutable default.